### ISSUE TITLE
Playback Error - Better way to handle it(?)

### ISSUE THREAD

**Issue by mathmarques** [OstlerDev/PopcornTV#143]
Hi!
First of all, this is an amazing idea! It works just perfect(and with subs!)!! (except for this issue =/)

Well, I was getting Playback error every time and my internet connection isn't that slow, 50Mb.
Every time, after 10~20 seconds after Playback Error appears, the console show: "Streamer: Ready to Stream, binding..."
Well, the torrent is working!
After look at the code, I saw this:
```
setTimeout(function(){
				if (!responded){
					xml.errorXML('Playback Error', 'Unable to download file fast enough, aborting stream attempt. Please try again with another quality/file.', function(errXML){
						response.write(errXML);
						response.end();
						responded = true;
						try{
							torrent.getStreamer.close();
						} catch(e) {}
					})
				}
			}, 59000)
```

Why 59000 mseconds?!
Must have another way to see if we have a torrent problem.
`torrent.on('error')` isn't enough?

Sometimes(Always) the torrent takes toooooo long to start, and 59000 mseconds isn't enough. After removing/changing this time limit, the Movie work just fine!

**Comment by OstlerDev** [comment]
The reason this is implemented is that 59000ms is 59 seconds, at 60 seconds the request on the Apple TV times out and would normally show "An error occurred", or something very similar. It was confusing a lot of users into thinking that there was something wrong with the program, so instead, one second before the Apple TV times out and throws an error we send it valid XML that explains what is happening.

Try starting up PopcornTV in debug mode, it will fully show you what exactly is happening in the background and how fast the torrent is actually downloading.

I have thought of better ways to do this, but they would require a large rewrite and as such I would like to save that for a version such as v0.1.7. Solely for the amount of time that is required to fix it when there are some other bugs and features that need to be added. That being said, if I can figure out an easy way to implement this then I will go for it, but for now, it will be staying how it is.

I hope that explains a bit behind the methodology of everything, I might need to go thru and comment everything heavily.

**Comment by OstlerDev** [comment]
This is something that you can see in Issue #2 which has been open since the start of the project.

**Comment by OstlerDev** [comment]
This will be fixed in #121
